speed-dreams/source-2.2.3/src/modules/graphic/ssggraph/grmain.cpp

```diff
@@ -536,7 +536,6 @@ initView(int x, int y, int width, int height, int /* flag */, void *screen)
     GfuiAddKey(screen, GFUIK_F11,      "TV Director View",  (void*)9, grSelectCamera, NULL);
 
     GfuiAddKey(screen, '5',            "Debug Info",        (void*)3, grSelectBoard, NULL);
-    GfuiAddKey(screen, '2',            "Driver Counters",   (void*)1, grSelectBoard, NULL);
```

Comment:
Add removed to CHANGES.txt

Reply:
It's already in there from a previous PR:
`changed initView(): removed keybinds for removed hud elements`